aptos-move/framework/aptos-stdlib/sources/math_fixed.move

```diff
@@ -0,0 +1,111 @@
+/// Standard math utilities missing in the Move Language.
+module aptos_std::math_fixed {
+    use std::fixed_point32;
+    use std::fixed_point32::FixedPoint32;
+
+    const EOVERFLOW: u64 = 1;
+
+    // Square root of fixed point number
+    public fun sqrt(x: FixedPoint32): FixedPoint32 {
+        let y = (fixed_point32::get_raw_value(x) as u128);
+        let prev_res = 0;
+        let res = y / 2;
+        while (res != prev_res) {
+            prev_res = res;
+            res = (res + (y << 32) / res) / 2;
+        };
+        fixed_point32::create_from_raw_value((res as u64))
+    }
+
+    // Exponent function with a precission of 6 digits.
+    public fun exp(x: FixedPoint32): FixedPoint32 {
+        fixed_point32::create_from_raw_value((exp_raw((fixed_point32::get_raw_value(x) as u128)) as u64))
```

Comment:
Can you break this into several statements with better variable names to improve readability?